index.bs

```diff
@@ -385,6 +414,113 @@ spec: workers; urlPrefix: https://www.w3.org/TR/workers/#
         {{VideoConfiguration/contentType}}, if possible. Otherwise, the UA should
         presume that the video stream cannot produce alpha channel information.
       </p>
+
+      <p>
+        <pre class='idl'>
+          enum HdrMetadataType {
+            "smpteSt2086",
+            "smpteSt2094-10",
+            "smpteSt2094-40"
+          };
+        </pre>
+
+        <p>
+          If specified, the <dfn for='VideoConfiguration' dict-member>hdrMetadataType</dfn> 
+          describes the capability to interpret HDR metadata of the specified type.
+        </p>
+
+        <p>
+          The {{HdrMetadataType}} values are:
+          <ul>
+            <li>
+              <dfn for='HdrMetadataType' enum-value>smpteSt2086</dfn>, representing
+              the static metadata type defined by [[SMPTE-ST-2086]].
+            </li>
+            <li>
+              <dfn for='HdrMetadataType' enum-value>smpteSt2094-10</dfn>, representing
+              the dynamic metadata type defined by [[SMPTE-ST-2094]].
+            </li>
+            <li>
+              <dfn for='HdrMetadataType' enum-value>smpteSt2094-40</dfn>, representing
+              the dynamic metadata type defined by [[SMPTE-ST-2094]].
+            </li>
+          </ul>
+        </p>
+      </p>
+
+      <p>
+	      <pre class='idl'>
+	        enum ColorGamut {
+	          "srgb",
+	          "p3",
+	          "rec2020"
+	        };
+	      </pre>
+	
+	      <p>
+	        The {{ColorGamut}} represents the color gamut supported by the UA. 
```

Comment:
Here {{ColorGamut}} links to the type definition, where hdrMetataType above links to the dict-member. Prefer the dict-member here and use definition structure similar to that from hdrMetadataType to link to an external def for the enum values.